uaclient/entitlements/repo.py

```diff
@@ -256,6 +256,13 @@ def install_packages(
         if not util.handle_message_operations(msg_ops):
             return
 
+        try:
+            self._update_main_repo()
```

Comment:
This update is required because the following:

The installed packages might depend on packages from the main archives, as for example, on a bionic instance that was not `apt-update`ed yet if one run:

```
pro attach <token> 
pro enable cis
```

then the pro client will install `usg-common` that depends on `libopenscap8` which comes from `-updates` and might not be yet on the apt cache.